src/NHibernate.Test/NHSpecificTest/EmbIdBidirectional/BiEmbIdRefEdEntity.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace NHibernate.Test.NHSpecificTest.EmbIdBidirectional
```

Comment:
Please use the `GHxxxx` subfolder naming when adding tests into `NHSpecificTest`.

Reply:
@fredericDelaporte 
Not really specific to this PR/test but a general opinion...
Why structure the test based on some github number? Isn't it better to give test namespaces a clear name? Personally I like that better.
You can always look up the git issue looking at the VCS history (assuming good commit messages including github number).

Reply:
@RogerKratz issue-number based namespaces allows instant lookup of the issue. It's like dictionary vs list.

Reply:
And `NHSpecificTest`is the place for issue-number based namespaces. Others are to be put outside of it. Those already there are a legacy.

So you can put them in a named folder, provided it is out of `NHSpecificTest`, and in most cases, in another suitable subfolder of the test project. (It has already many subfolders, so in most cases a suitable one should already exist.) Some bug cases can even be directly added to already existing fixtures in folders other than `NHSpecificTest`, when that makes sense.

Reply:
Ok.

(
Still I prefer working in projects where I can run a subset/namespace of the test suite to verify the area I'm currently working on. If there is no logical structure of the tests, you pretty much have to run _all_ the tests over-and-over again during your work and not just as a last verification in the end.
)